docker/.env

```
@@ -176,6 +185,7 @@ MF_CERTS_SIGN_CA_KEY_PATH=/etc/ssl/certs/ca.key
 MF_CERTS_SIGN_HOURS_VALID=2048h
 MF_CERTS_SIGN_RSA_BITS=2048
 MF_CERTS_VAULT_HOST=http://vault:8200
+MF_CERTD_INSTANCE_ID=
 
```

Since we `MF_CERTS` prefix for other environment variables , I suggest you to use the same here 
```suggestion
MF_CERTS_INSTANCE_ID=

```